templates/cli/lib/commands/pull.js.twig

```diff
@@ -163,6 +193,14 @@ const pull = new Command("pull")
         helpWidth: process.stdout.columns || 80
     });
 
+pull
+    .command("all")
+    .description("Push all resource.")
+    .option(`--all`, `Flag to pull all functions`)
+    .option(`--yes`, `Flag to confirm all warnings`)
+    .action(actionRunner(pullResources));
+
```

Comment:
Not sure if this should be a new command. Have you considered using just an option `--all` to the base pull command itself? So it would be 

```sh
appwrite pull project
appwrite pull functions
appwrite pull --all
``` 
??

Reply:
The reason this was done, when using `appwrite pull --all` we won't get the error message like this for example
```sh
> appwrite pull function
error: unknown command 'function`
(Did you mean functions?)

```
As everything is the first command

Reply:
Alright 👍